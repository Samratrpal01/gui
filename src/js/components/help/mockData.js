export const helpProps = {
  docsVersion: undefined,
  isHosted: false,
  menderVersion: 'master',
  isEnterprise: true,
  tenantCapabilities: {},
  tokens: [],
  versions: {
    repos: {
      mender: '3.3.0',
      'mender-artifact': '3.8.0',
      'mender-binary-delta': '1.3.1',
      'mender-cli': '1.8.0',
      'mender-configure-module': '1.0.4',
      'mender-connect': '2.0.2',
      'mender-convert': '3.0.0',
      'mender-gateway': '1.0.0',
      'monitor-client': '1.2.0'
    },
    releaseDate: '01-01-2003'
  }
};
