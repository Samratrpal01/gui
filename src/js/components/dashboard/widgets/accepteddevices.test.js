import React from 'react';

import { undefineds } from '../../../../../tests/mockData';
import { render } from '../../../../../tests/setupTests';
import AcceptedDevices from './accepteddevices';

describe('AcceptedDevices Component', () => {
  it('renders correctly', async () => {
    const { baseElement } = render(<AcceptedDevices offlineThreshold={{ interval: 24, intervalUnit: 'hours' }} />);
    const view = baseElement.firstChild.firstChild;
    expect(view).toMatchSnapshot();
    expect(view).toEqual(expect.not.stringMatching(undefineds));
  });
});
