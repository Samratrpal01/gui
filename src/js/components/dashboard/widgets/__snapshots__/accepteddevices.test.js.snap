// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AcceptedDevices Component renders correctly 1`] = `
<div
  class="widget "
>
  <div
    class="flexbox widgetHeader"
  >
    <div
      class="flexbox center-aligned"
    >
      Accepted devices 
    </div>
  </div>
  <div
    class="flexbox column widgetMainContent"
  >
    <div
      class="counter"
    />
  </div>
</div>
`;
